Resources/Scripts/Gui/StartupScreen/ConfigViewTabs.as

```diff
@@ -467,15 +467,18 @@ namespace spades {
         StartupScreenConfigView @configViewOpenAL;
         StartupScreenConfigView @configViewYSR;
 
+        StartupScreenAudioOpenALEditor @editOpenAL;
+
         private ConfigItem s_audioDriver("s_audioDriver");
         private ConfigItem s_eax("s_eax");
+        private ConfigItem s_openalDevice("s_openalDevice");
 
         StartupScreenAudioTab(StartupScreenUI @ui, Vector2 size) {
             super(ui.manager);
             @this.ui = ui;
             @helper = ui.helper;
 
-            float mainWidth = size.x - 250.f;
+            float mainWidth = size.x - 120.f;
```

Comment:
This narrow width of the help pane causes word-wrap fallback in some Slavic languages. Does the original width cause any functional issues other than long device names running over the combo box's width?

Reply:
Not at all, I did not think of other speaking languages indeed.